data/categoryRepository.go

```diff
@@ -125,7 +124,6 @@ LEFT JOIN measurement_series ON measurement_series.measurement_id = data_list_me
 LEFT JOIN series ON series.id = measurement_series.series_id
 LEFT JOIN feature_toggles ON feature_toggles.name = 'filter_by_quarantine'
```

Comment:
Good point. I'll remove the join. The other instances of the quarantine filter should remain. We just want the category summaries to be independent of the quarantined status of the associated series.